scipy/stats/stats.py

```diff
@@ -8121,6 +8121,237 @@ def brunnermunzel(x, y, alternative="two-sided", distribution="t",
+        integer, taken as the number of samples available (strictly positive)
+        quantile : float The quantile for which we want to compute the
+        confidence interval. Must be strictly between 0 and 1. confidence :
+        float The desired confidence level of the confidence interval. Must be
+        strictly between 0 and 1.
+    type : {'one-sided', 'two-sided'}, optional
+        Defines the type of confidence interval computed. Default is
+        'one-sided'.
+          * 'one-sided' :
+          computes the best possible one-sided confidence intervals
+          (both lower and upper bounds) for the given quantile.
+          * 'two-sided' :
+          computes a two-sided confidence interval by combination of
+          two one-sided intervals. E.g., a 90% two-sided interval is
+          computed by combining two 95% one-sided intervals.
+    Returns
+    -------
+    LB : float or int or `None` value or index of the lower bound of
+        * the right-open one-sided confidence interval (default),
+        * a two-sided confidence interval
+        `None` is returned when there are not enough samples to compute
+        the confidence interval with the desired level of confidence.
+    UB : float or int  or None value or index of the upper bound of
+        * the left-open one-sided confidence interval (default),
+        * a two-sided confidence interval
+        `None` is returned when there are not enough samples to compute
+        the confidence interval with the desired level of confidence.
+    Notes
+    -----
+    Two-sided confidence intervals are not guaranteed to be optimal. I.e., there
+    may exist a tighter interval that may contain the quantile of interest with
+    probability larger than the confidence level. These intervals may be found
+    by exhaustive search, which we do not do for efficiency reasons.
+    Without further assumption on the samples (eg, the nature of the underlying
+    distribution), the one-sided intervals are optimally tight.
+    References
+    ----------
+    .. [1] W. R. Thompson, "On Confidence Ranges for the Median and Other
+       Expectation Distributions for Populations of Unknown Distribution Form,"
+       The Annals of Mathematical Statistics, vol. 7, no. 3, pp. 122-128, 1936,
+       Accessed: Sep. 18, 2019. [Online]. Available:
+       https://www.jstor.org/stable/2957563.
+    .. [2] H. A. David and H. N. Nagaraja, "Order Statistics in Nonparametric
+       Inference" in Order Statistics, John Wiley & Sons, Ltd, 2005, pp.
+       159-170. Available:
+       https://onlinelibrary.wiley.com/doi/10.1002/0471722162.ch7.
+    .. [3] N. Hutson, A. Hutson, L. Yan, "QuantileNPCI: Nonparametric
+       Confidence Intervals for Quantiles," R package,
+       https://cran.r-project.org/package=QuantileNPCI
+    Examples
+    --------
+    >>> from scipy.stats import confint_quantile x = [2, 8, 3, 6, 4, 1, 5, 9, 7]
```

Comment:
```suggestion
    >>> from scipy.stats import confint_quantile
    >>> x = [2, 8, 3, 6, 4, 1, 5, 9, 7]
```
And please add blank lines before and after the code snippets.

From [numpydoc standards](https://numpydoc.readthedocs.io/en/latest/format.html):
![image](https://user-images.githubusercontent.com/6570539/120056095-f9436f80-bfee-11eb-9141-d4bac791864c.png)